doc/api.rst

```diff
@@ -10,7 +10,7 @@ API
    stanc
    StanModel
 
-:ref:`StanFit4model <StanFit4model>` instances are also documented on this page.
+:ref:``StanFit4model <StanFit4model>`` instances are also documented on this page.
```

Comment:
This doesn't seem like the right syntax. https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html

Reply:
StanFit4model <StanFit4model>

Is that supposed to be a link to StanFit4model docs?

Reply:
I think the current format is correct

    :ref:`StanFit4model`